api/capi/doc/nnstreamer_capi.md

```diff
@@ -0,0 +1,315 @@
+# nnstreamer capi
```

Comment:
For this doc, it is ok; In general Linux, this is NNStreamer C-API.
However in the context of Tizen (for github.com/Samsung/...), this is called "Machine Learning Inference C-API".

Reply:
I'll change the name at tizen-docs